docs/public-networks/how-to/send-transactions.md

```diff
@@ -59,3 +59,18 @@ compares the characteristics of both calls.
 | Does not consume gas                                           | Requires gas                                                                                                                  |
 | Synchronous                                                    | Asynchronous                                                                                                                  |
 | Returns the value of a contract function available immediately | Returns transaction hash only. A block might not include all possible transactions (for example, if the gas price is too low). |
+
+## Use wallets for key management
+
+Besu doesn't support key management inside the client. Use:
+
+* [EthSigner](http://docs.ethsigner.consensys.net/en/latest/) with Besu to provide access to your
```

Comment:
And Web3Signer?

Reply:
As far as I understand, Web3Signer is focused on signing consensus layer data and works for some custom eth1 infrastructure, but doesn't work with Besu by default.

Reply:
Depends on the signing scheme but web3signer supports secp256k1 which can be used with Besu.